### PR TITLE
Stun guns now don't ignore armor.

### PR DESCRIPTION
**Pull request by TheShown911**
## About The Pull Request

Stun guns now take account burn(energy) armor and as compensation they get a whopping 1 burn damage.
hallos more like hell
left(naked)(slight delay on refreshing hallos) / right(techno voidsuit)(30% energy) - NT SP "Counselor" used for tests

## Why It's Good For The Game

Making armor choice apply to stun guns is cool, also no more dual wielding 60 hitscan damage a hit with small guns.

## Changelog
:cl: TheShown
balance: stun beams don't bypass armor anymore
/:cl:

### CODE REVIEW

**Comment by Kurgis** [comment]
Doesn't this reduce the stun damage of all ranged stun weapons?

**Comment by TheShown911** [comment]
> Doesn't this reduce the stun damage of all ranged stun weapons?

The `nodamage` var makes the beam do no damage (no 1 burn damage on the test subjects) so the damage actuallly stays the same as the test showed (30 damage a hit on unarmored).
demig

**Review by genessee596 — APPROVED** [review]
Delegate stamp of approval.

Makes armor more useful - paincrit is a death sentence in combat, no matter how much armor you have. This makes armor more useful for keeping away pain, to some extent.

Still needs maintainer approval.

**Review by TorinoFermic — APPROVED** [review]

**Pull request merged**
